tools/ci_build/github/pai/buildtrainingdoc.sh

```diff
@@ -0,0 +1,23 @@
+# This script must be executed from this folder.
+
```

Comment:
For each bash file, please add "#!/bin/bash" as the first line.

Reply:
Sure, n/p.